apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -147,6 +154,10 @@ const BookingPage = ({
 
   const locationInfo = (type: LocationType) => locations.find((location) => location.type === type);
   const loggedInIsOwner = eventType?.users[0]?.name === session?.user?.name;
+  const guestListEmails = !isDynamicGroupBooking
+    ? booking?.attendees.slice(1).map((attendee) => attendee.email)
+    : [];
+
```

Comment:
Same as below, but since now we are using more than once we pass it values to a const